3rdParty/Radon/Radon/include/Key.hpp

```diff
@@ -23,7 +23,7 @@ namespace radon
 
 		float getFloatValue();
 
-		void setValue(float & value);
+		void setValue(const float & value);
```

Comment:
Nice.

Reply:
It's better to pass primitive types by value (just `float`)

Reply:
I guess that can be done.